Dockerfile

```diff
@@ -1,5 +1,5 @@
 # First, get the elixir dependencies within an elixir container
-FROM hexpm/elixir:1.11.4-erlang-23.3.4.18-alpine-3.16.2 AS elixir-builder
+FROM hexpm/1.17.3-erlang-27.1-alpine-3.19.4 as elixir-builder
```

Comment:
question: does this work for you? I got `ERROR: failed to solve: hexpm/1.17.3-erlang-27.1-alpine-3.19.4: failed to resolve source metadata for docker.io/hexpm/1.17.3-erlang-27.1-alpine-3.19.4:latest` when I tried `docker build .` locally

Reply:
Sorry I am still faffing with this! yes the elixir bit is missing which I've added locally, but the issue now is getting these errors:

```
Error loading shared library libstdc++.so.6: No such file or directory (needed by /home/document_viewer/erts-15.1/bin/beam.smp)
Error loading shared library libgcc_s.so.1: No such file or directory (needed by /home/document_viewer/erts-15.1/bin/beam.smp)
```

despite having added:

```
RUN apk add --update libstdc++6 libstdc++ libssl1.1 ...
```
to the file...

Reply:
Think I managed to fix this now.